trackers/browser-tracker/test/tracker.test.ts

```diff
@@ -276,3 +277,24 @@ describe('Activity tracker behaviour', () => {
     expect(secondPageId).toBe(extractPageId(ppl));
   });
 });
+
+describe('API', () => {
+  it('getQueueName does return the queue name', () => {
```

Comment:
Probably just nest this under another `describe('getQueueName', () => {})`?

Nit: s/does return/returns/